src/main/java/org/kohsuke/github/GHRepositorySearchBuilder.java

```diff
@@ -160,6 +196,33 @@ public GHRepositorySearchBuilder sort(Sort sort) {
         STARS, FORKS, UPDATED
     }
 
+    /**
+     * The enum Fork.
+     */
+    public enum Fork {
+        ALL_INCLUDING_FORKS("true"), FORKS_ONLY("only");
+
+        private String filterMode;
+        Fork(String mode) {
+            this.filterMode = mode;
+        }
+
+        public String toString() {
+            return filterMode;
+        }
+    }
+
+    /**
+     * The enum Visibility.
+     */
+    public enum Visibility {
+        PUBLIC, INTERNAL, PRIVATE;
+
+        public String toString() {
+            return name().toLowerCase(Locale.ENGLISH);
+        }
+    }
+
```

Comment:
Use `GHRepository.Visibility` instead? 

https://github.com/hub4j/github-api/blob/9978678f276619e2bf81e6542a596e74ce1dc0cc/src/main/java/org/kohsuke/github/GHRepository.java#L729-L740

Reply:
@bitwiseman  yeah, I agree, we do not need duplications.
What is expected behavior if we use GHRepository.Visibility.UNKNOWN? Should we just ignore this value in GHRepositorySearchBuilder?

Reply:
@t0m4uk1991
Doh, `UNKNOWN` should have JavaDoc explaining usage.   

If passed, it would be reasonable to ignore or maybe even throw an exception with a message saying "UKNOWN cannot be passed into this method."

Reply:
in the current implementation, I just ignore this param if it equals to GHRepository.Visibility.UNKNOWN, also I added some explanation in javadocs